types/lsp.ts

```diff
@@ -0,0 +1,3 @@
+export { TextDocument } from 'vscode-languageserver-textdocument'
```

Comment:
We should be able to use `export type` syntax here to signify that `TextDocument` is not supposed to be used downstream as an object with side-effect, but only provides a type.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html

Reply:
(fixed)